stellarphot/tests/test_core.py

```diff
@@ -963,10 +963,36 @@ def test_catalog_from_vizier_search_apass():
 
 
 @pytest.mark.remote_data
-def test_catalog_from_vizier_search_vsx():
+@pytest.mark.parametrize("location_method", ["coord", "header", "wcs"])
+def test_catalog_from_vizier_search_vsx(location_method):
     # Do a cone search with a small enough radius to return exaclty one star,
     # DQ Psc, which happens to already be in the test data.
     coordinate = SkyCoord(ra=359.94371 * u.deg, dec=-0.2801 * u.deg)
+
+    if location_method == "coord":
+        center = coordinate
+    else:
+        # We need a WCS for each of these methods, so make that first.
+        # The projection doesn't matter because we only need the center
+        # coordinate. The code below is more or less copied from
```

Comment:
"copied from " where? I want to know....

Reply:
Added it :)